src/validator.js

```diff
@@ -24,6 +25,13 @@ export default class Validator {
     this.collector = new Collector();
   }
 
+  addToCollector(message) {
+    var messageType = message.severity.charAt(0).toUpperCase() +
+                      message.severity.slice(1);
+    console.log(message);
```

Comment:
Could probably remove this - though at some point it would be nice to have some configurable logging - we should add an issue to add something with debug levels.